en/3-meeds.md

```diff
@@ -1,80 +1,81 @@
+The Meeds Association has minted 5 Million MEED (5% of the total supply) at the start to acquire software IP from the funding members, pay for token emission legal fees, and hosting services. 
+Then, it locked 4.5 Million in a vault contract with a linear vesting over four years.
 
-The rest is part of the __Meeds DAO funds__ and will be used to pay for services and launch incentives such as airdrops and referral campaigns.
+Since then, a smart contract called **the __Token Factory__ performs minting of **MEED tokens at a constant rate**. 
 
+The minting rate is a parameter of the Token Factory, set at *10 MEED per minute* at the start. A vote of the DAO may change this rate in the future. 
+At such rate, the Token Factory mints around 5 Million MEED yearly (10*60*24*365); thus, the minting horizon is nearly 20 years.
 
 ## 3.2. Minting allocation
 
-Minted Meeds are allocated to encourage involvement of the three categories of stakeholders - the users, the builders and the investors:
+The Token Factory also rules the allocation of minted MEED tokens to  three categories of stakeholders :
 
-- The users (users and Deed holders/renters) get Meeds through engagement rewards (mintium model).
-- Builders (members of the association and community) get Meeds through the association funds to build, operate and improve the WoM.
-- Investors mint Meeds through renting and buying of liquidity pools and staking Meeds.
+- **Builders**: Contributors of the Meeds DAO that help build and run the WoM
+- **Users**: Members in all communities of the WoM earn MEED as rewards for their contributions
+- **Investors**: The Meeds DAO incentivizes MEED token owners to stake MEED or provide liquidity for the Meeds economy 
 
-At the beginning, the builders get 30% of the allocation. The 70% are shared between the users and the investors. This allocation proportion will be reviewed on a regular basis, through the Meeds association DAO vote.
+Each category of stakeholders receives a share of the MEED supply, which is subject to votes by the DAO.
 
 ## 3.3. Users – minting through engagement
 
-Part of the continuously minted Meeds is allocated to the engagement minting channel. The distribution is done weekly. At any given week end, the engagement index is calculated so that Hubs leaders and Deed owners can claim their rewards the following week and get their Meeds. Unclaimed Meeds stay in the fund and are added to the next week’s distribution.
-The model is described in detail in the previous chapter (part 2.6).
+Part of the continuously minted Meeds is allocated to sponsor user engagement through the Mintium model (see 2.6). A global engagement index is calculated periodically across all Hubs of the WoM to allocate MEED rewards to the Hubs. Mintium rewards are claimable by Hub leaders and Deed owners.
 
-## 3.4. Builders – Meeds association DAO and community of builders
+## 3.4. Builders – Earn for building the WoM
```

Comment:
I seem to be a little confused. Comparing and contrasting the two statements "Meeds association DAO and community of Builders" and "Earn for building the WoM" do not relate to the same information. 
While the first talks about the Builders of Meeds DAO and community broadly;, the latter narrows down only to the benefits.

Reply:
agreed.  I'm going to simplify like this : 
3.3. Users
3.4. Builders
3.3. Investors